crates/core_simd/src/ord.rs

```diff
@@ -0,0 +1,222 @@
+use crate::simd::{intrinsics, LaneCount, Mask, Simd, SimdElement, SupportedLaneCount};
+
+/// Parallel `PartialOrd`.
+pub trait SimdPartialOrd {
+    /// The mask type returned by each comparison.
+    type Mask;
+
+    /// Test if each lane is less than the corresponding lane in `other`.
+    #[must_use = "method returns a new mask and does not mutate the original value"]
+    fn simd_lt(self, other: Self) -> Self::Mask;
```

Comment:
I think we should name the new methods `lanes_*` rather than `simd_*` for consistency and leave it to the rename-to-`elementwise_*` PR to resolve that.

Reply:
Regarding "elementwise", I'm not sure functions themselves should necessarily have that prefix.  I was just thinking our terminology in the docs.  I figured `simd_` might be a good, terse way of indicating that you're not calling regular `Ord`.

Reply:
i still stsnd by my consistency argument, if you want `simd_*` to be the prefix, then we should rename all `lanes_*` functions to `simd_*`

Reply:
These traits entail all of the functions with the `lanes_` prefix

Reply:
(testing github bug with reviews, please ignore)

Reply:
ah, ok.